pkg/manager/impl/execution_manager_test.go

```diff
@@ -198,6 +199,17 @@ func getMockRepositoryForExecTest() repositories.RepositoryInterface {
 
 func TestCreateExecution(t *testing.T) {
 	repository := getMockRepositoryForExecTest()
+	labels := admin.Labels{
+		Values: map[string]string{
+			"label3": "3",
+			"label2": "1", // common label, will be dropped
```

Comment:
do we want project labels to override individual labels?

Reply:
They do not override, see the code: https://github.com/lyft/flyteadmin/pull/129/files#diff-fc047e54b9dd82ca7c89ac9b32cb07b3R1375

Reply:
ah thank you for clarifying!